integration-tests/contracts/ethereum_contracts.go

```diff
@@ -1934,12 +1936,14 @@ type LegacyEthereumOffchainAggregatorV2 struct {
 
 // OCRv2Config represents the config for the OCRv2 contract
 type OCRv2Config struct {
-	Signers               []common.Address
-	Transmitters          []common.Address
-	F                     uint8
-	OnchainConfig         []byte
-	OffchainConfigVersion uint64
-	OffchainConfig        []byte
+	Signers                  []common.Address
+	Transmitters             []common.Address
+	F                        uint8
+	OnchainConfig            []byte
+	TypedOnchainConfigLegacy i_keeper_registry_master_wrapper_2_1.KeeperRegistryBase21OnchainConfig
+	TypedOnchainConfig       i_automation_registry_master_wrapper_2_2.AutomationRegistryBase22OnchainConfig
```

Comment:
these are probably better as `OnChainConfigV21` and `OnChainConfigV22` etc

Reply:
done